docs/web_ssr/styling.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 5
+---
+
+# Styling
+
+## Global variables
+
+This library mainly uses global CSS variables for its style (mainly the colors). You can easily adapt the theme of the library to your website's design simply by overriding those variables.
+
+```html
+<style>
+  :host {
+    --miam-ds-color-primary: blue;
+    --miam-ds-color-background-primary: #FCFCFC;
+
+    .miam-ds-text {
+      font-weight: 500;
+    }
```

Comment:
Le font-weight est lié aux classes .weight-XXX (ex: `.miam-ds-text.weight-xl`) du coup c'est pas l'exemple le + pertinent
```suggestion
    .miam-ds-text {
      font-family: "Roboto";
    }
```
À ce sujet, dans une prochaine itération de la doc ça pourrait être bien de documenter un peu plus en profondeur miam-ds (avec des sections pour les boutons, les textes, les tabs etc, un peu dans le style de la démo de miam-ds ou dans le style des maquettes)